src/SFML/Window/Unix/GlxContext.cpp

```diff
@@ -269,9 +272,6 @@ void GlxContext::display()
 ////////////////////////////////////////////////////////////
 void GlxContext::setVerticalSyncEnabled(bool enabled)
 {
-    // Make sure that extensions are initialized
-    ensureExtensionsInit(m_display, DefaultScreen(m_display));
```

Comment:
What was the reasoning for this being here in the first place, or why can it be removed now?